### DISCUSSION TITLE
😫 Stuck Setting Up Project Locally

### DISCUSSION THREAD

**Discussion by ashrafchowdury** [openstatusHQ/openstatus#533] [Q&A]
Hi everyone,

I'm having trouble setting up this project locally and I believe the issue lies with the installation guide provided in the documentation. It seems the references to the `libsql/sqld` repository are outdated, as the server has moved its location.

**I'm using Windows 11, that's why I used WSL(Windows Subsystem For Linux) to set the database**

Here's what I've tried:

* Followed the installation steps in the documentation verbatim.
* Tried searching for the old `sqld` repository at the linked address, but it's no longer available.
* Looked for alternative installation instructions, but haven't found anything specific to this project's needs.

### Screenshots

After running `brew install sqld-beta`

![image](https://github.com/openstatusHQ/openstatus/assets/87828904/936f9a09-b1a0-4a33-bd7d-1da7ac37bafc)

Tried to run  `brew install sqld`

![image](https://github.com/openstatusHQ/openstatus/assets/87828904/dcfac715-9f69-49a7-a3bd-30567b4f95bb)

----

`libsql/sqld` repository

![image](https://github.com/openstatusHQ/openstatus/assets/87828904/3639f1a1-91db-4492-9b7b-e692f013d0af)

### My system configuration

```
System:
    OS: Windows 11 10.0.22621
    CPU: (8) x64 AMD Ryzen 5 3400G with Radeon Vega Graphics    
    Memory: 461.96 MB / 5.95 GB
  Binaries:
    Node: 20.9.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 10.1.0 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.10.2 - ~\AppData\Roaming\npm\pnpm.CMD
  Browsers:
    Edge: Chromium (119.0.2151.97)
    Internet Explorer: 11.0.22621.1
```

**Comment by thibaultleouay** [comment]
@ashrafchowdury  Sorry for the delay, 
Have you tried setting up the platform on windows?
